src/server/scripts/World/conversation_scripts.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2008-2017 TrinityCore <http://www.trinitycore.org/>
+ * Copyright (C) 2008-2017 TrinityCore <https://www.trinitycore.org/>
```

Comment:
lul, im pretty sure its http in all headers

Reply:
should be changed on next copyright update